bitnami/postgresql-ha/Chart.yaml

```
@@ -28,4 +28,3 @@ sources:
   - https://github.com/bitnami/bitnami-docker-postgresql
   - https://www.postgresql.org/
 version: 7.2.0
```

A new version appeared in the meantime
```suggestion
version: 7.3.0
```